_includes/head/custom.html

```diff
@@ -6,3 +6,7 @@
     Prism.manual = true;
 </script>
 <script type="module" src="https://unpkg.com/rapidoc/dist/rapidoc-min.js"></script>
+
+{% if site.baseurl %}
+<div id="banner">This is a non official version of the documentation. Click <a href="/">here</a> to return to the official docs.</div>
```

Comment:
"non official" stimmt ja nicht, oder? Offiziell ist sie ja schon, nur eben eine Beta. Weiß nicht ob Beta und nicht offiziell dieselbe Definition haben.

Reply:
Stimme dir zu, hast du ne bessere Idee?

Reply:
"This is documentation for version 2 of Enmeshed, which is currently in non-stable beta status. For documentation of the current version, see these docs." (docs ist ein Link)